block_rich_text.go

```diff
@@ -312,64 +312,76 @@ func NewRichTextSectionTeamElement(teamID string, style *RichTextSectionTextStyl
 type RichTextSectionUserGroupElement struct {
 	Type        RichTextSectionElementType `json:"type"`
 	UsergroupID string                     `json:"usergroup_id"`
+	Style       *RichTextSectionTextStyle  `json:"style,omitempty"`
 }
 
 func (r RichTextSectionUserGroupElement) RichTextSectionElementType() RichTextSectionElementType {
 	return r.Type
 }
 
-func NewRichTextSectionUserGroupElement(usergroupID string) *RichTextSectionUserGroupElement {
+func NewRichTextSectionUserGroupElement(usergroupID string, style *RichTextSectionTextStyle) *RichTextSectionUserGroupElement {
```

Comment:
This breaks backward-compatibility, so if you want to use this immediately, maybe divide this PR into another one.

Reply:
(If there is nothing wrong with including next minor release, you don't divide.)

Reply:
Thanks for the review.
You are right about breaking backward compatibility.  This PR should only be a comma modification of `omitempty`, and the style addition changes should be created as a new separate PR.

Reply:
PR was changed to comma change only.